src/datepicker/datepicker-navigation.ts

```diff
@@ -55,4 +55,14 @@ export class NgbDatepickerNavigation {
   @Output() select = new EventEmitter<NgbDate>();
 
   constructor(public i18n: NgbDatepickerI18n) {}
+
+  onClickPrev(event: MouseEvent) {
+    (event.target as HTMLElement).focus();
```

Comment:
This won't work if you click on the internal `<span>` element, unfortunately. Could we change the implemnetation to something like this ?

```html
<button #prev (click)="onNavigate(prev, navigation.PREV)"></button>
```

```ts
onNavigate(button: HTMLButtonElement, direction: NavigateEvent) {}
```

Reply:
@peterblazejewicz @maxokorokov this is a bit overkill, I would suggest to switch from `event.target` to `event.currentTarget`

```typescript
onClickPrev(event: MouseEvent) {
    (event.currentTarget as HTMLElement).focus();
}
```

Reply:
thanks folks for the comments, the detail's has been reworked!